contrib/hadoop/src/org/xtreemfs/common/clients/hadoop/XtreemFSFileSystem.java

```diff
@@ -168,20 +168,32 @@ public void initialize(URI uri, Configuration conf) throws IOException {
                     trustedCertificatesFileContainer, conf.getBoolean("xtreemfs.ssl.authenticationWithoutEncryption",
                             false), false, sslProtocolString, null);
         }
-
+        
+        // Initialize default options
+        Options xtreemfsOptions = new Options();
+        xtreemfsOptions.setMetadataCacheSize(0);
+        
+        // Use the native client and use async writes if requested.
         ClientFactory.ClientType clientType = ClientFactory.ClientType.JAVA;
         if (conf.getBoolean("xtreemfs.jni.enabled", false)) {
             clientType = ClientFactory.ClientType.NATIVE;
+            
+            String libraryPath = conf.get("xtreemfs.jni.library-path");
```

Comment:
ah ja macht sinn das für das project konsistent zu halten. ich war da vorher glaube ich zuviel in yarn-default.xml unterwegs, die snake-cases verwenden und hab gar nich mehr geschaut was unserer convention ist. ich pass das gleich nochmal überall an